src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+  const propertiesArray = sourceString.split(';');
+
+  propertiesArray.map((element) => {
```

Comment:
Method `map` returns new array. In this case would be better to use `forEach` method